src/test/java/io/gravitee/common/util/URIUtilsTest.java

```diff
@@ -192,10 +194,45 @@ public void test22() {
         MultiValueMap<String, String> parameters = URIUtils.parameters("?foo1&k=v&foo");
         Assertions.assertEquals(3, parameters.size());
         Assertions.assertTrue(parameters.containsKey("foo1"));
-        Assertions.assertEquals(null, parameters.get("foo1").get(0));
+        Assertions.assertNull(parameters.get("foo1").get(0));
         Assertions.assertTrue(parameters.containsKey("foo"));
-        Assertions.assertEquals(null, parameters.get("foo").get(0));
+        Assertions.assertNull(parameters.get("foo").get(0));
         Assertions.assertTrue(parameters.containsKey("k"));
         Assertions.assertEquals("v", parameters.get("k").get(0));
     }
+
+    @Test
+    public void shouldBeAbsoluteWithHttp() {
+        assertThat(URIUtils.isAbsolute("http://api.gravitee.io/echo")).isTrue();
+    }
+
+    @Test
+    public void shouldBeAbsoluteWithAnyProtocol() {
+        assertThat(URIUtils.isAbsolute("any://api.gravitee.io/echo")).isTrue();
+    }
+
+    @Test
+    public void shouldBeAbsoluteWithHttps() {
+        assertThat(URIUtils.isAbsolute("https://api.gravitee.io/echo")).isTrue();
+    }
+
+    @Test
+    public void shouldNotBeAbsoluteWithInvalidProtocol() {
+        assertThat(URIUtils.isAbsolute("a://api.gravitee.io/echo")).isFalse();
+    }
+
+    @Test
+    public void shouldNotBeAbsoluteWithoutProtocol() {
+        assertThat(URIUtils.isAbsolute("api.gravitee.io/echo")).isFalse();
+    }
+
+    @Test
+    public void shouldNotBeAbsoluteWithPath() {
+        assertThat(URIUtils.isAbsolute("/echo")).isFalse();
+    }
+
+    @Test
+    public void shouldNotBeAbsoluteWithNull() {
+        assertThat(URIUtils.isAbsolute(null)).isFalse();
+    }
```

Comment:
Maybe a [Parametrized](https://www.baeldung.com/parameterized-tests-junit-5) test could be interesting but I don't know if `gravitee-common` is Junit5 compatible 🙈

Reply:
Yes it is JUnit5. I've replaced the tests with parameterized test.